net-dialup/freeradius/freeradius-3.0.17.ebuild

```diff
@@ -214,7 +219,14 @@ src_install() {
 	systemd_newtmpfilesd "${FILESDIR}"/freeradius.tmpfiles freeradius.conf
 	systemd_dounit "${FILESDIR}"/freeradius.service
 
-	find "${ED}" \( -name "*.a" -o -name "*.la" \) -delete || die
+	# Remove all default sites under site-enables.
+	# Administrator install all sites when configure freeradius.
+	rm "${D}"/etc/raddb/sites-enabled/{inner-tunnel,default}
+	# Remove part of modules that require a configuration
+	# from mods-enabled directory.
+	rm "${D}"/etc/raddb/mods-enabled/{cache_eap,digest,eap,mschap,ntlm_auth}
```

Comment:
missing die

Reply:
why doesn't to permit an installation that works ? But then I didn't think that if a user upgrade package and he has a site active we remove it. My fault. I remove it. Thanks